dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/ProcessDefinitionLog.java

```diff
@@ -39,6 +41,11 @@
     @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss", timezone = "GMT+8")
     private Date operateTime;
 
+    /**
+     * execution type
+     */
+    private ProcessExecutionTypeEnum executionType;
```

Comment:
I suggest not to put this field in the processdefinitionlog. The inheritance method has been obtained